pytest_rts/plugin.py

```diff
@@ -1,14 +1,24 @@
 """Code for pytest-rts plugin logic"""
 import logging
 import os
+import sqlite3
 import pytest
 from pytest_rts.pytest.init_phase_plugin import InitPhasePlugin
 from pytest_rts.pytest.normal_phase_plugin import NormalPhasePlugin
 from pytest_rts.pytest.update_phase_plugin import UpdatePhasePlugin
-from pytest_rts.utils.common import update_mapping_db
-from pytest_rts.utils.db import DatabaseHelper, DB_FILE_NAME
-from pytest_rts.select import get_tests_and_data_committed, get_tests_and_data_current
 from pytest_rts.utils.git import get_current_head_hash
+from pytest_rts.utils.selection import (
+    get_tests_and_data_committed,
+    get_tests_and_data_current,
+)
+from pytest_rts.utils.mappinghelper import MappingHelper
+from pytest_rts.utils.testgetter import TestGetter
+
+DB_FILE_NAME = "mapping.db"
+INIT_REQUIRED = not os.path.isfile(DB_FILE_NAME)
```

Comment:
Try to make sure that the results of `import my_module` do not depend on the environment. The checks such as "does this file exist?" should be done not at load-time, but at run-time.